Slam/Queue/APIManagedQueue.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public class APIManagedQueue: QueueManager {
-    public let queueView: QueueView
+    public var queueView: QueueView?
```

Comment:
It'd be nice if this were not an Optional. Based on our use of the class, it looks like we would always be passing in a `QueueView`.